build/references.py

```diff
@@ -171,8 +172,33 @@ def get_standard_citatation(citation, cache, override):
 template = jinja2.Template(converted_text)
 converted_text = template.render(**stats)
 
+all_sections_file = gen_dir.joinpath('all-sections.md')
+
+#FIXME: Stubbing metadata:
+title_text = "Manubot Rootstock: Manuscript Title"
```

Comment:
I realize that no metadata should be hardcoded. I added a [stub](https://en.wikipedia.org/wiki/Test_stub) to avoid significantly altering project layout without some direction. Where in the build process should I add the call to metadata.py (the file where I am going to move this to)? Should metadata be conditionally supported in the build process?